src/maintainer/adding_pkgs.rst

```diff
@@ -49,17 +54,18 @@ Step-by-step Instructions
    should be downloadable as an archive (.tar.gz, .zip, .tar.bz2, .tar.xz)
    or tagged on GitHub, to ensure that it can be verified. (For further
    detail, see :ref:`tarballs_no_repos`).
-#. Fork the `example recipes
-   <https://github.com/conda-forge/staged-recipes/tree/master/recipes>`_
-   repository.
-#. Create a new branch from the staged-recipes ``master`` branch.
-#. Within your forked copy, generate a new folder in the recipes subdirectory
-   and copy the `meta.yml
+#. Fork and clone the `staged-recipes
+   <https://github.com/conda-forge/staged-recipes>`_
+   repository from GitHub.
+#. Checkout a new branch from the staged-recipes ``main`` branch.
+#. Through CLI, cd inside the 'staged-recipes/recipes' directory.
+#. Within your forked copy, create a new folder in the recipes folder for your package (i.e, ``...staged-recipes/recipes/<name-of-package>``)
+#. Copy `meta.yml
```

Comment:
I think this is a typo, right?

```suggestion
#. Copy `meta.yaml
```

Reply:
Ah! Yes. Thanks for catching that.

Reply:
Btw, what I usually do in this step is:

```console
# unix
cp -r recipes/example recipes/<package_name>
# windows
xcopy /h /e /i "recipes\example\" "recipes\<package_name>"
```

, so this step and the previous are the same one. But it's essentially the same, consider it a tip!